packages/upload-api/src/blob/list.js

```diff
@@ -0,0 +1,15 @@
+import * as Server from '@ucanto/server'
+import * as Blob from '@web3-storage/capabilities/blob'
+import * as API from '../types.js'
+
+/**
+ * @param {API.BlobServiceContext} context
+ * @returns {API.ServiceMethod<API.BlobList, API.BlobListSuccess, API.Failure>}
+ */
+export function blobListProvider(context) {
```

Comment:
Nit: I have avoided spec-ing blob/list because I'm not sure what should it return exactly. Broadly speaking I've been thinking that all our list operations should be client side and simply provide view over relevant receipts.